src/server.cpp

```diff
@@ -1811,13 +1811,13 @@ void Server::SendMovePlayer(session_t peer_id)
 	assert(sao);
 
 	NetworkPacket pkt(TOCLIENT_MOVE_PLAYER, sizeof(v3f) + sizeof(f32) * 2, peer_id);
-	pkt << sao->getBasePosition() << sao->getPitch() << sao->getYaw();
+	pkt << sao->getBasePosition() << sao->getLookPitch() << sao->getYaw();
```

Comment:
I was confused by sao.getYaw() here but remember that PlayerSAO inherits from UNITSAO. UNITSAO has contains the function for setYaw(), not in PlayerSAO. It is getting the player model yaw.